app/controllers/users/mfa_selection_controller.rb

```diff
@@ -30,6 +30,15 @@ def update
       redirect_back(fallback_location: two_factor_options_path, allow_other_host: false)
     end
 
+    # @api private
+    def two_factor_options_form
```

Comment:
moved this so I could call `contorller.two_factor_options_form` instead of having to throw a `.send` on there, the `# @api private` comes from YARD, it's just a label, not a visibility modifier